services/testing-framework/longhorn.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 const (
-	maxLonghornCheck = 240 // max allowed time for pods of longhorn-system to be ready [seconds]
-	sleepSecPods     = 10  // seconds for one cycle of longhorn checks (the node and pod checks)
+	maxLonghornCheck = 10 * 60 // max allowed time for pods of longhorn-system to be ready [seconds]
```

Comment:
Do we see change in time taken to deploy all longhorn-system pods with increase in number of nodes?

Reply:
Yes, I saw increased time to deploy pods into the cluster, however, I wouldn't say it was due to an increased number of nodes, more. Maybe the lack of compute power on the nodes chosen